src/Native/Runtime/gcenv.h

```diff
@@ -174,7 +174,12 @@ class EEConfig
     int     GetGCtraceFac  ()               const { return 0; }
     int     GetGCprnLvl    ()               const { return 0; }
     bool    IsGCBreakOnOOMEnabled()         const { return false; }
+#ifdef CORERT
+    // CORERT-TODO: remove this
```

Comment:
Add a link to issue? https://github.com/dotnet/corert/issues/913 should be covering this.